pylint/checkers/classes.py

```diff
@@ -926,9 +926,12 @@ def _check_unused_private_functions(self, node: nodes.ClassDef) -> None:
                     continue
                 if isinstance(attribute.expr, nodes.Name) and attribute.expr.name in (
                     "self",
+                    "cls",
```

Comment:
Another option could be:
`"cls" if attribute.scope().type == "classmethod" else "self",` instead of listing both, "cls" and "self".

Reply:
I think it's clear, let me know what you think about factorizing this check with the othe rone, but this could be merged right now :)